api/resolvers/notifications.js

```diff
@@ -91,6 +91,18 @@ export default {
               AND "Item"."userId" <> $1 AND "Item".created_at <= $2
             ${await filterClause(me, models)}
             ORDER BY "sortTime" DESC
+            LIMIT ${LIMIT}+$3)
+          UNION DISTINCT
```

Comment:
This `union distinct` probably won't do any elimination since the `type` is `FollowActivity`, not `Reply`.

Reply:
You're right, it won't.

We'll need to `group by` the item id to make them distinct.

Reply:
Let me know if you want me to address this in this PR, or if it should be lumped in with the general de-dupe effort mentioned at the root-level of this PR.